src/app/utils/modal-routes.enum.ts

```diff
@@ -1,5 +1,6 @@
 export enum ModalRoutes {
-    ToCatalog = 'Go to catalog',
-    QuizzesCatalog = 'Quizzes catalog',
-    Finish = 'Finish',   
+    Statistics = '/statistics',
+    QuizzesCatalog = '/quizzes-catalog',
+    Finish = '/finish',   
+    Refresh = 'refresh'
```

Comment:
looks like Refresh can be deleted

Reply:
I added this field because I thought it looked better with a variable

Reply:
ok. let it be